packages/web-app-files/src/helpers/resources.js

```diff
@@ -273,13 +273,18 @@ export function aggregateResourceShares(
   token
 ) {
   if (incomingShares) {
+    shares = addSharedWithToShares(shares)[0]
```

Comment:
This commit here can be omitted now. https://github.com/owncloud/web/pull/6717 got merged.